<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>358</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>499</width>
    <height>358</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>499</width>
    <height>358</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LANchat</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>251</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_con">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_subnet">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="inputMask">
    <string>000.000.000.000;_</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Subnet:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_nobroadcast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Do not send broadcast</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_netmask">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="inputMask">
    <string>00;_</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>302</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nickname">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>LANchatter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_discon">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nickname:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_nickname</tabstop>
  <tabstop>lineEdit_subnet</tabstop>
  <tabstop>lineEdit_netmask</tabstop>
  <tabstop>pushButton_con</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>checkBox_nobroadcast</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>pushButton_discon</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>OnSend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_con</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>OnConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_discon</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>OnDisconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnSend()</slot>
  <slot>OnConnect()</slot>
  <slot>OnDisconnect()</slot>
 </slots>
</ui>
